code/game/objects/items/devices/object-in-a-box.dm

```diff
@@ -0,0 +1,29 @@
+/obj/item/device/object_in_a_box
```

Comment:
It's just an item needed for atmoslathe to dispense /obj/structure due to it runtiming to hell due to missing vars. This item is just the product of atmoslathe that,when used, deploys the "master" object(currently only used for chemtank.)